### PR TITLE
Helm tests: collect container logs

### PR DESCRIPTION
**Pull request by azhavoro**
### Checklist

- [x] I submit my changes into the `develop` branch
- [ ] I have added a description of my changes into the [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
- [ ] I have updated the documentation accordingly
- [ ] I have added tests to cover my changes
- [ ] I have linked related issues (see [GitHub docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary
  ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
  [cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning),
  [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and
  [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.

### CODE REVIEW

**Review by kirill-sizov — APPROVED** [review]
LGTM, but I think we can also collect logs from our worker containers

**Comment by azhavoro** [comment]
> LGTM, but I think we can also collect logs from our worker containers

added
https://github.com/opencv/cvat/actions/runs/4691808547

**Comment by nmanovic** [comment]
@azhavoro , there are some problems with help tests in the PR. Should we resolve them before merging?

File: .github/workflows/helm.yml [hunk 1]
```diff
@@ -83,3 +83,22 @@ jobs:
...
+      if: failure()
+      env:
+          LOGS_DIR: "${{ github.workspace }}/rest_api_testing"
+      run: |
+        mkdir ${LOGS_DIR}
+        kubectl logs $(kubectl get pods -l component=server -o 'jsonpath={.items[0].metadata.name}') >${LOGS_DIR}/cvat_server.log
+        kubectl logs $(kubectl get pods -l component=worker-import -o 'jsonpath={.items[0].metadata.name}') >${LOGS_DIR}/cvat_worker_import.log
+        kubectl logs $(kubectl get pods -l component=worker-export -o 'jsonpath={.items[0].metadata.name}') >${LOGS_DIR}/cvat_worker_export.log
+        kubectl logs $(kubectl get pods -l component=worker-webhooks -o 'jsonpath={.items[0].metadata.name}') >${LOGS_DIR}/cvat_worker_webhooks.log
+        kubectl logs $(kubectl get pods -l app.kubernetes.io/name=traefik -o 'jsonpath={.items[0].metadata.name}') >${LOGS_DIR}/traefik.log
```
Comment (Marishka17) [inline .github/workflows/helm.yml:97]: Maybe need to use `for` loop here

**Pull request merged**